interfaces/Portal/src/app/services/programs-service-api.service.ts

```diff
@@ -165,7 +165,7 @@ export class ProgramsServiceApiService {
           })
           .sort(
             (a: PaymentData, b: PaymentData) =>
-              // Sort by payment-id (as the back-end doesn't do that)
+              // Sort by payment-id (as the back-end doesn't do that) - Is this still needed now?
```

Comment:
Saw this, don't think we need it now, but also don't think its worth the effort refactoring the old portal...